docs/reference/rest-api/root.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[rest-api-root]]
+=== Root API
+++++
+<titleabbrev>Version API</titleabbrev>
+++++
+
+The Elasticsearch API's base url returns its basic build, 
+version, and cluster statistics. 
+
+[source,console]
+--------------------------------------------------
+GET /
+--------------------------------------------------
+
+[[rest-api-root-prereq]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the
+`monitor`, `manage`, or `all`
+<<privileges-list-cluster,cluster privilege>> to use this API.
+
+[role="child_attributes"]
+[[rest-api-root-response-body]]
+==== {api-response-body-title}
+
+
+`name` ::
+The responding <<node-name,Node's `name`>>.
+
+`cluster_name` ::
+The responding <<cluster-name,Cluster's `name`>>.
+
+`cluster_uuid` ::
+The responding Cluster's `uuid` as confirmed by 
+<<cluster-state,Cluster State>>.
+
+`version` ::
+(object) 
+Contains version statistics about the build.
++ properties of `version`
+[%collapsible%open]
+====
+`number` ::
+Version number of responding 
+https://www.elastic.co/downloads/past-releases#elasticsearch[Elasticsearch release].
+
+`build_flavor` ::
+Build flavor, e.g. `default`.
+
+`build_type` ::
+Build type corresponding to how 
+<<install-elasticsearch,Elasticsearch was installed>, 
+e.g. `docker`, `rpm`, `tar`.
+
+`build_hash` ::
+Elasticsearch's Git commit's SHA hash.
+
+`build_date` ::
+Elasticsearch's Git commit's date.
+
+`build_snapshot` ::
+If Elasticsearch's build was from a snapshot.
+
+`lucene_version` ::
+Version number of Elasticsearch's 
+<<https://archive.apache.org/dist/lucene/java/,underlying Lucene software>>.
+
+`minimum_wire_compatibility_version` ::
+Minimum node version with which the responding node can 
+communicate. Also minimum  version from which you can perform 
+a <<rolling-upgrades,Rolling Upgrade>>.
+
+`minimum_index_compatibility_version` ::
+Minimum index version with which the responding node can read 
+from disk.
+====
+
+[[rest-api-root-response-example]]
+==== {api-examples-title}
+
+The API returns the following response: 
+
+[source,console-result]
+----
+{
+  "name": "instance-0000000000",
+  "cluster_name": "my_test_cluster",
+  "cluster_uuid": "5QaxoN0pRZuOmWSxstBBwQ",
+  "version": {
+    "build_date": "2024-02-01T13:07:13.727175297Z",
+    "minimum_wire_compatibility_version": "7.17.0",
+    "build_hash": "6185ba65d27469afabc9bc951cded6c17c21e3f3",
+    "number": "8.12.1",
+    "lucene_version": "9.9.2",
+    "minimum_index_compatibility_version": "7.0.0",
+    "build_flavor": "default",
+    "build_snapshot": false,
+    "build_type": "docker"
+  },
+  "tagline": "You Know, for Search"
+}
+----
+// TESTRESPONSE[s/"name" : "instance-0000000000"/"name" : $body.name/]
+// TESTRESPONSE[s/"cluster_name" : "my_test_cluster"/"cluster_name" : $body.cluster_name/]
+// TESTRESPONSE[s/"cluster_uuid" : "5QaxoN0pRZuOmWSxstBBwQ"/"cluster_uuid" : $body.cluster_uuid/]
```

Comment:
You'll need to replace most of the test response here. The test failure is because all of these numbers will vary based on the platform we test on, version number of the branch, etc.

https://gradle-enterprise.elastic.co/s/j3rbi5ewtqoyc/console-log/task/:docs:yamlRestTest?page=1#L253